roles/nsx_node/tasks/main.yml

```
@@ -0,0 +1,119 @@
+---
+- name: NSX | Download CNI rpm
+  get_url:
+    url: "{{ nsx_cni_url }}"
+    dest: /tmp/cni.rpm
+  tags: nsx_node
+
+- name: NSX | Install CNI
+  yum:
+    name: /tmp/cni.rpm
+    state: present
+  tags: nsx_node
+
+- name: NSX | Download OVS
+  get_url:
+    url: "{{ nsx_ovs_url }}"
+    dest: /tmp/ovs.rpm
+  when: nsx_node_type != 'BAREMETAL'
+  tags: nsx_node
+
+- name: NSX | Download OVS kmod
+  get_url:
+    url: "{{ nsx_kmod_ovs_url }}"
+    dest: /tmp/kmod_ovs.rpm
+  when: nsx_node_type != 'BAREMETAL'
+  tags: nsx_node
+
+- name: NSX | Install OVS
+  yum:
+    name: /tmp/ovs.rpm
+    state: present
+  when: nsx_node_type != 'BAREMETAL'
+  tags: nsx_node
+
+- name: NSX | Install OVS Kernel Module
+  yum:
+    name: /tmp/kmod_ovs.rpm
+    state: present
+  when: nsx_node_type != 'BAREMETAL'
+  tags: nsx_node
+
+- name: NSX | Fix Openvswitch wrong SELinux label
+  command: /usr/sbin/restorecon -R -v /run/openvswitch/
+  tags: nsx_node
+
+- name: NSX | Restart OVS Kernel Module
+  command: /etc/init.d/openvswitch force-reload-kmod
+  tags: nsx_node
+
+- name: NSX | Start OVS service
+  systemd:
+    name: openvswitch
+    state: started
+  tags: nsx_node
+
+- name: NSX | Create OVS bridge
+  openvswitch_bridge: 
```

`
  57:22     error    trailing spaces  (trailing-spaces)
`
```suggestion
  openvswitch_bridge:
```